nodes/object_nodes/select_mesh_verts.py

```diff
@@ -27,7 +27,7 @@
 class SvSelectMeshVerts(bpy.types.Node, SverchCustomTreeNode):
     ''' Select vertices of mesh objects '''
     bl_idname = 'SvSelectMeshVerts'
-    bl_label = 'Select Object Vertices'
+    bl_label = 'Select Vertices of Mesh'
```

Comment:
This node really does place mesh elements of `scene.Objects` in a selected state, not sure if this is an improvement